### PR TITLE
README.md : Mention projects.coala.io

### PR DESCRIPTION
**Pull request by Warewolf96**
Added projects.coala.io in README.md

Closes https://github.com/coala/projects/issues/142

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -3,7 +3,7 @@
 A website that showcases projects! **If you want to use it and it's not exactly
 fitting your purpose, please do make upstream changes. We expect this project
 to be of use to others and we're happy to make it more generic so it's easier
-to adapt to other organizations.**
+to adapt to other organizations.** To view live projects visit projects.coala.io
```
Comment (hemangsk) [inline README.md]: 👍  It'll be good to link it, check this out https://github.com/adam-p/markdown-here/wiki/Markdown-Cheatsheet#links

It'll look like
```
To view live projects visit [some-text](projects.coala.io)
```

**Comment by gaocegege** [comment]
coala.io/commit

```diff
- README.md : Mention projects.coala.io
- Added projects.coala.io in README.md
+ README.md: Mention projects.coala.io
+ Add projects.coala.io in README.md
```

**Comment by sims1253** [comment]
unack b8b847e

Can you add coala as the creator of the project? Something like "this thing was built by the coala community, you can see the page life here: link"

**Comment by Warewolf96** [comment]
@sims1253 can u please review again..

File: README.md [hunk 2]
```diff
@@ -3,7 +3,8 @@
 A website that showcases projects! **If you want to use it and it's not exactly
 fitting your purpose, please do make upstream changes. We expect this project
 to be of use to others and we're happy to make it more generic so it's easier
-to adapt to other organizations.**
+to adapt to other organizations.** This was built by coala community, to view
+live projects visit [this link.](projects.coala.io)
```
Comment (sils) [inline README.md]: "this link" isn't a useful link text, why not `[projects.coala.io](https://projects.coala.io)`?

**Comment by sils** [comment]
please read coala.io/commits and do a rebase

**Comment by hemangsk** [comment]
@Warewolf96 can you please rebase it 😃

**Comment by Warewolf96** [comment]
@hemangsk I guess this should do it

**Comment by Mariatta** [comment]
Sorry, this go out-of-date again :(
Please rebase.

**Comment by sils** [comment]
Maybe someone can just cherry pick it

**Comment by Warewolf96** [comment]
now it's up-to-date

**Comment by meetmangukiya** [comment]
Ack a9ec9f7

**Comment by Warewolf96** [comment]
Is this ready for merge?

**Comment by sils** [comment]
@rultor merge

**Comment by sils** [comment]
please read coala.io/commit carefully though and use colons like you do in normal english ;)

**Pull request merged**